app/services/metadata_service.rb

```diff
@@ -39,8 +39,22 @@ def local_attribution_list
       values
     end
 
-    def format_list
-      ['Cartographic', 'Data', 'Image', 'Notated music', 'Text', 'Video', 'Sounding recordings-nonmusical']
+    def resource_type_list
```

Comment:
I wonder if we could, eventually, turn this into a rake task or something that would use RDF.rb functionality to run through a list of URI's, fetch the graphs, and extract the labels. Not for this PR of course, but when we move to building out the new application.

Similar to the [gist](https://gist.github.com/mcritchlow/e36e841b9e6a019a971c6ce7858cb555/edit) I mocked up earlier when we were discussing subjects, but grabbing the labels instead 

While I doubt this list would change often, in other contexts the lists might. So not having them hard-coded would be interesting. Or, even allowed curators to manage the lists via the UI.